apps/root/src/frame/index.tsx

```diff
@@ -144,6 +145,7 @@ const AppFrame = ({ config: { wagmiClient }, initialChain }: AppFrameProps) => {
                                 path="/:chainId/positions/:positionVersion/:positionId"
                                 element={<PositionDetail />}
                               />
+                              <Route path=":chainId/vaults/:strategyGuardianId" element={<StrategyGuardianDetail />} />
```

Comment:
I know this might sound like a break in patterns, but I woudl really make this `/earn/:chainId/vaults/:strategyGuardianId`

I hate how the position one ended up in

Reply:
Me too, haha. I believe Earn can be an opportunity to improve other DCA routes (that can be optimized, aside from the position route). I can create a task for it if you agree

Reply:
Please do :). Because we also have to keep backwards compatibility